Lombiq.Tests.UI.Samples/Tests/JavascriptTests.cs

```diff
@@ -0,0 +1,67 @@
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Services.GitHub;
+using System.IO;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Lombiq.Tests.UI.Samples.Tests;
+
+// Suppose you want to write UI tests in Javascript. Why would you want to do that? Unlikely if you are an Orchard Core
+// developer, but what if the person responsible for writing the tests is not? In the previous training section we
+// discussed using a separate frontend server, with mention of technologies using Node.js. In that case the frontend
+// developers may be more familiar with Javascript so it makes sense to write and debug the tests in Node.js so they
+// don't have to learn different tools and tech stacks just to create some UI tests.
+public class JavascriptTests : UITestBase
+{
+    public JavascriptTests(ITestOutputHelper testOutputHelper)
+        : base(testOutputHelper)
+    {
+    }
+
+    // Using this approach you only have to write minimal C# boilerplate, which you can see below.
+    [Fact]
+    public Task ExampleJavascriptTestShouldWork() =>
+        ExecuteTestAfterSetupAsync(context =>
+        {
+            // Don't forget to mark the script files as "Copy if newer", so they are available to the test. It's best to
+            // include something like the following in your csproj file:
+            // <None Update="Tests\*.mjs" CopyToOutputDirectory="PreserveNewest" />
+            var workingDirectory = "Tests";
+            var scriptPath = Path.Join(workingDirectory, "test.mjs");
```

Comment:
These two seem like values that can be computed by default apart from the test script name, and used as is in the majority of cases, instead of having to pass them in. Also in `Sandbox()`.

Reply:
In OFFI I did that with an additional "setup and test" method that derived the path from the caller's namespace. But that kind of stuff is too opinionated and unnecessary complexity.

Reply:
_Tests_ here refers to the build output directory, i.e. the test's execution directory, which is not necessarily the directory of the current/JS code file. How about while keeping the option to specify the working directory open, default that to the current directory, and recommend copying mjs files to the root of the output directory instead? Then, this could be:

```c#
return context.SetupSeleniumAndExecuteJavascriptTestAsync(_testOutputHelper, /*scriptName parameter*/"test.mjs");
```